tests/bud.bats

```diff
@@ -59,6 +59,36 @@ load helpers
   rm -rf ${TESTSDIR}/bud/use-layers/mount
 }
 
+@test "bud with --layers, multistage, and COPY with --from" {
+  mkdir -p ${TESTSDIR}/bud/use-layers/uuid
+  uuidgen > ${TESTSDIR}/bud/use-layers/uuid/data
+  mkdir -p ${TESTSDIR}/bud/use-layers/date
+  date > ${TESTSDIR}/bud/use-layers/date/data
+
+  buildah bud --signature-policy ${TESTSDIR}/policy.json --layers -t test1 -f Dockerfile.multistage-copy ${TESTSDIR}/bud/use-layers
+  run buildah --debug=false images -a
+  [ $(wc -l <<< "$output") -eq 6 ]
+  [ "${status}" -eq 0 ]
+  buildah bud --signature-policy ${TESTSDIR}/policy.json --layers -t test2 -f Dockerfile.multistage-copy ${TESTSDIR}/bud/use-layers
+  run buildah --debug=false images -a
+  [ $(wc -l <<< "$output") -eq 7 ]
+  [ "${status}" -eq 0 ]
+
+  uuidgen > ${TESTSDIR}/bud/use-layers/uuid/data
+  date > ${TESTSDIR}/bud/use-layers/date/data
+  buildah bud --signature-policy ${TESTSDIR}/policy.json --layers -t test3 -f Dockerfile.multistage-copy ${TESTSDIR}/bud/use-layers
```

Comment:
It would probably be good to add a 'test4' that dropped the '--layers' param from the buildah bud command.